packages/wonder-blocks-core/util/get-clickable-behavior.js

```diff
@@ -14,7 +14,7 @@ import {withRouter} from "react-router-dom";
 import ClickableBehavior from "../components/clickable-behavior.js";
 
 function isExternalUrl(url: string) {
-    return /^https?:\/\//i.test(url);
+    return url.indexOf("//") >= 0;
```

Comment:
We should have tests for this.  Currently `/foo//bar` would be incorrectly identified as being external.  Maybe we can do:
```
/^(https?:)?\/\//i.test(url);
```

Reply:
Sure. I'll update the check and add test as well.